forms-mit/opt-in.html

```diff
@@ -1,68 +1,132 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<!--[if lte IE 9]><html class="no-js lte-ie9" lang="en"><![endif]-->
+<!--[if !(IE 8) | !(IE 9) ]><!-->
+<html lang="en-US" class="no-js">
+<!--<![endif]-->
 <head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
   <title>MIT Authors' Opt-in Open Access license : MIT Libraries</title>
-  <link href="/css/header-semantic.css" rel="stylesheet" type="text/css">
-  <link href="/css/wp-libraries.css" rel="stylesheet" type="text/css">
-  <!--#include virtual="/includes/base_scripts.html" -->
+
+  <!--#include virtual="/mitlib-style/inc/assets-site-head.html" -->
+
+  <!-- form specific stuff -->
```

Comment:
This stylesheet is fine to be inline for now, but when @hllavina gets to her form work my hope is that this will be abstracted out to a general forms stylesheet, maybe at something like `/forms/style.css` or similar.